Sources/Purchasing/OfferingsManager.swift

```diff
@@ -366,6 +405,39 @@ private extension OfferingsManager {
             return PreviewProductType(packageType: packageType) ?? .default
         }
     }
+
+    func trackPurchaseStartedIfNeeded(trackDiagnostics: Bool) {
+        if #available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *),
```

Comment:
I don't think the `#available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *)` check is needed here

Reply:
Hmm I believe it is needed since `trackOfferingsStarted` is actually marked as available only in those versions?

Reply:
Ohhh you're right, again 😄. I didn't need this in my previous PRs because my where I added the events was already restricted to these versions. But yes, you're totally right. Sorry again 🙏